jobs/expirynotify/scheduler.go

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package expirynotify
+
+import (
+	"time"
+
+	"github.com/mattermost/mattermost-server/v5/app"
+	"github.com/mattermost/mattermost-server/v5/model"
+)
+
+const (
+	SCHED_FREQ_MINUTES = 10
+)
+
+type Scheduler struct {
+	App *app.App
+}
+
+func (m *ExpiryNotifyJobInterfaceImpl) MakeScheduler() model.Scheduler {
+	return &Scheduler{m.App}
+}
+
+func (scheduler *Scheduler) Name() string {
+	return "NotifyExpiryScheduler"
+}
+
+func (scheduler *Scheduler) JobType() string {
+	return model.JOB_TYPE_EXPIRY_NOTIFY
+}
+
+func (scheduler *Scheduler) Enabled(cfg *model.Config) bool {
+	return true
```

Comment:
I assume this could be change in the future

Reply:
Good point.  I was disabling the feature with a config check in `initJobs` but this looks like the intended way to enable/disable jobs.